front/app/modules/commercial/insights/admin/containers/Insights/Edit/InputsTable/Actions.tsx

```diff
@@ -149,6 +154,24 @@ const Actions = ({
     }
   };
 
+  const approveSuggestedCategories = async () => {
+    setProcessingBulkApprove(true);
+
+    for (const input of selectedInputs) {
+      try {
+        await addInsightsInputCategories(
+          viewId,
+          input.id,
+          input.relationships.suggested_categories.data
+        );
+      } catch {
+        // do nothing
+      }
+    }
+    setProcessingBulkApprove(false);
+  };
+
+  console.log(processingBulkApprove);
```

Comment:
Don't forget to remove this `console.log`

Reply:
Thanks for catching this 👍